src/pycatcher/diagnostics.py

```diff
@@ -115,12 +115,15 @@ def build_seasonal_plot(df):
 
 def build_iqr_plot(df):
```

Comment:
Updated some config which didn't change the existing outcome on the notebook.
<img width="937" alt="image" src="https://github.com/user-attachments/assets/2bc33d4c-869c-4e1c-9884-4cfc45d89806">